app/models/task_management.rb

```diff
@@ -35,7 +35,7 @@ def self.all_notifications_for(user_type, id)
   def self.update_all_notifications_as_seen(user)
     query = user.user_type == "tasker" ? "tasker_id" : "taskee_id"
     attribute = query.gsub("id", "notified")
-    where(query => user.id).where(attribute => false).
+    where(query => user.id).where(attribute => false).where(paid: true).
```

Comment:
It will not be right to use scopes here as these methods collect arguments. We can, but it won't be the rails way. Scopes are best used when there are no passed arguments.

[Rails docs](http://guides.rubyonrails.org/active_record_querying.html#passing-in-arguments)